### ISSUE TITLE
new firmware 2.30.20 seems not work

### ISSUE THREAD

**Issue by NickeryXu** [acecilia/OpenWRTInvasion#136]
Router: Mi 4A Gigabit edition
Firmware Version: 2.30.20
System: Windows 10
OpenWrtInvasion version: 0.0.8

no exception in scripts, but telnet failed; I choose local TCP, looks like build a local tcp server, so that shouldn't be a network problem; I have already enable telnet client; the only reason I can find is this firmware version might change api or something, realy needs help :(
![失败截图](https://user-images.githubusercontent.com/42167910/173213217-fd7a6c57-f6fa-4359-acd1-ad8155fcc325.png)

**Comment by devidlis123** [comment]
Hello, mine is the same version. No solution?

**Comment by NickeryXu** [comment]
> Hello, mine is the same version. No solution?
Yup, finally I change my device to Redmi AX6S, got a snopshot OpenWRT and that works fine

**Comment by borekon** [comment]
Duplicate from #135

**Comment by acecilia** [comment]
Thanks @borekon 🙌 Closing as duplicate
